server/integration/build/assembly-fat-server.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I want all distro assemblies (servers, sources, etc) to be relegated to the distribution module and not scattered everywhere. Why is this different from the server zip generated in distribution ?

Reply:
this is no longer in use and I'll remove it. But, the only difference I can see, is the copy of the licenses.